explorer/src/pages/Attestations/table/components/SortByDate/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { ArrowDownUp } from "lucide-react";
```

Comment:
is there any point in this component if it already exists https://github.com/Consensys/linea-attestation-registry/blob/dev/explorer/src/components/SortByDate/index.tsx